### DISCUSSION TITLE
Comparison with ELRS MAVLink?

### DISCUSSION THREAD

**Discussion by uxduck** [olliw42/mLRS#233] [Q&A]
ExpressLRS has a mavlink feature: https://www.expresslrs.org/software/mavlink/. I'd love to know the difference with mLRS. Personally I just wanna know the practical difference but implementation/technical details is probably interesting/useful for me and others too.

**Comment by olliw42** [comment]
while this might be an interesting question, please understand that the mLRS team does NOT consider it its job to provide a comparison to other projects. The mLRS team in fact would find that largely inappropriate. We believe we have published the data and information needed to evaluate mLRS. You can join general discussion channels for getting the info you seek.

I think your question has been answered comprehensively, so I close.

**Reply by uxduck** [reply]
I'm a bit surprised with your response to be honest. It seems like you've taken the question personally. Why is the question inappropriate - is there some background I'm missing? I assumed you as the creator would know best about the differences. This also gives you the opportunity to share your comparative advantage and vision.

Don't you think your users want to know the differences, so they can make the best decision?

For anyone curious about my question: After some research, I came across some useful content on the ExpressLRS community discord. Just search for mLRS there, e.g.

> mLRS does 16ch (but their resolution and update rate are severely limited)

**Comment by olliw42** [comment]
I have not said the question is inappropriate but that us doing a comparison is in my opinion inappropriate. That's my moral compass at least.
